README.md

```diff
@@ -0,0 +1,139 @@
+# Accommodation Service Web Application
+
+## Introduction
+
+Welcome to the Accommodation Service Web Application!
+This project was inspired by the need for a comprehensive platform that allows users to browse,
+purchase, and manage bookings online. It aims to provide a seamless experience for both users
+and administrators. Users can browse bookings by status, receive notifications about actions,
+and create new bookings, while administrators have the ability to manage the accesses and accommodations.
+
+## Technologies and Tools Used
+
+This project leverages a range of modern technologies and tools to ensure robust performance and security:
+
+- **Spring Boot**: For building the backend of the application.
+- **Spring Security**: To handle authentication and authorization.
+- **Spring Data JPA**: For database operations.
+- **Swagger**: To document and test the APIs.
+- **Liquibase**: For database version control.
+- **Docker**: To containerize the application for easy deployment.
+- **MapStruct**: For object mapping.
+- **Stripe**: For taking payment from users.
+- **Telegram**: For sending notifications to users.
+
+## Features
+
+### User Functionalities
+- **Create Booking**: Users can create and manage their bookings.
+- **User Manage**: Users can check info about their profiles and update it.
+- **View Accommodations**: Users can view all accommodations or specific by id.
+
+### Admin Functionalities
+- Admins can update roles for users
+- Admins can browse all bookings by status
+- Admins can create, update and delete accommodations
+
+## Setup Instructions
+
+To set up the project locally, follow these steps:
+
+### Prerequisites
+
+Ensure you have the following software installed:
+
+- **Java 21**
+- **Docker Version 4.30.0 (149282)**
+- **Maven 3.9.5**
+
+### Database Configuration
+
+You can use either Postgres or H2 database.
+For simplicity, the following instructions will use H2.
+If you prefer Postgres, ensure you update the necessary configuration in `application.properties`.
+
+### Environment Variables
+
+Create a .env file in the root directory of your project with the following content:
```

Comment:
It's better to specify that this is an example and that it's necessary to add your own variables.